src/app/home/home.js

```diff
@@ -0,0 +1,47 @@
+$(document).ready((function ($) {
+    'use strict';
+
+    function getFormData($form) {
```

Comment:
Make sure this is moved to `app.js` - I left a comment in `auth.js`

Reply:
Ok

Reply:
Resolved.